pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -79,10 +79,15 @@ mkDerivation rec {
     "-DCOMPILE_QML=ON" # see https://github.com/Nheko-Reborn/nheko/issues/389
   ];
 
-  preFixup = lib.optionalString voipSupport ''
-    # add gstreamer plugins path to the wrapper
-    qtWrapperArgs+=(--prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "$GST_PLUGIN_SYSTEM_PATH_1_0")
-  '';
+  preFixup = ''
+    qtWrapperArgs+=( \
+      --prefix QT_PLUGIN_PATH : ${lib.makeSearchPath "/lib/qt5/plugins" [ qtbase ]} \
```

Comment:
This shouldn't be necessary: the `QT_PLUGIN_PATH` variable is already set by the `wrapQtAppsHook`.